src/openrct2/network/NetworkBase.cpp

```diff
@@ -46,7 +46,7 @@ using namespace OpenRCT2;
 // It is used for making sure only compatible builds get connected, even within
 // single OpenRCT2 version.
 
-#define NETWORK_STREAM_VERSION "1"
+#define NETWORK_STREAM_VERSION "2"
```

Comment:
@ZehMatt do you know why it's giving me a warning here?

Reply:
You touched on code where clang tidy has a recommendation. No need to change it, we can ignore and move on. Or if you want you can make another commit where you replace this network version macro by a constexpr.